pkgs/development/compilers/picat/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, gcc49, gnumake }: 
+
+stdenv.mkDerivation {
+  name = "picat-1.9-4";
+
+  src = fetchurl {
+    url = http://picat-lang.org/download/picat19_src.tar.gz;
+    sha256 = "0wvl95gf4pjs93632g4wi0mw1glzzhjp9g4xg93ll2zxggbxibli";
+  };
+
+  buildInputs = [ stdenv gcc49 gnumake ];
+
+  ARCH = if stdenv.system == "i686-linux" then "linux32"
```

Comment:
Either way, it's conceivable that this could be built on other architectures soon.  Picat includes Makefiles for cygwin and OSX, but I don't have access to those for testing.